docs/user-guide/introspection-and-debugging.md

```diff
@@ -35,53 +35,67 @@ spec:
         - containerPort: 80
 ```
 
+Copy this to a file *./my-nginx-dep.yaml*
```

Comment:
My preference is to check in the yaml file and use curl.  That is blocked on coming up a sane way to group the guides and related config together.  It may be as simple as moving the .md to a directory and renaming the file.  I prefer John's widget to the cat approach.